### ISSUE TITLE
creating the environment and modifications to genscore.py

### ISSUE THREAD

**Issue by CLG68** [sc8668/GenScore#13]
Hi,

1) It took me some time to get a working environment.  For sure, you should remove this from your read.me file:

conda create --prefix xxx --file ./requirements_conda.txt      
pip install -r ./requirements_pip.txt

Here is how I was able to create the environment:

conda create -n genscore python=3.8.11
conda activate genscore
conda install nvidia/label/cuda-11.5.2::cuda-toolkit
conda install mdanalysis==2.0.0 -c conda-forge
conda install rdkit==2021.03.5 -c conda-forge
pip install prody==2.1.0
conda install numpy==1.20.3 -c conda-forge
pip install torch==1.11.0+cu115 torchvision==0.12.0+cu115 torchaudio==0.11.0  --extra-index-url https://download.pytorch.org/whl/cu115
pip install torch-geometric==2.0.3
pip install torch-scatter==2.0.9 -f https://pytorch-geometric.com/whl/torch-1.11.0+cu115.html
pip install seaborn==0.11.2
pip install pandas==1.3.2
pip install matplotlib==3.4.3
pip install scipy==1.6.2
pip install scikit-learn==0.24.2
conda install conda-forge::pytorch_sparse
pip install torch_sparse==0.6.15 -f https://pytorch-geometric.com/whl/torch-1.11.0+cu115.html
conda install openbabel==3.1.0 -c conda-forge
pip install joblib==1.0.1

2) in genscore.py, you hard-coded the link to openbabel libraries but you did not indicate it in your Read.me file

**_#you need to set the babel libdir first if you need to generate the pocket
#os.environ["BABEL_LIBDIR"] = "/home/shenchao/.conda/envs/my3/lib/openbabel/3.1.0"_**

You could replace this code by:

**default_babel_libdir = os.path.join(os.getenv("CONDA_PREFIX", ""), "lib", "openbabel", "3.1.0")
os.environ["BABEL_LIBDIR"] = os.getenv("BABEL_LIBDIR", default_babel_libdir)**

so it is not hard-coded anymore.  However, it will still be linked to a variable... CONDA_PREFIX

3) I get deprecation warnings from MDAnalysis, Bio.pairwise2, and pkg_resources.  Maybe other versions would solve the problem.

Best,
Christian
